docs/guidelines.md

```diff
@@ -113,7 +113,30 @@ ngeo.exampleDirective = function(…) {
       // …
     },
     controllerAs: 'ctrl',
+    // …
+  });
+```
+
+But if you bind your directive to the controller and you declare your attribute
+as a two ways binding attrribute, you can use the `.` notation. See the example
```

Comment:
Also note that in the [colorpicker code](https://github.com/camptocamp/ngeo/blob/master/src/directives/colorpicker.js#L39-L41) the properties for `scope` are not given as strings.